pkg/cmd/stack/destroy.go

```diff
@@ -148,18 +147,16 @@ func destroyDeployments(ctx context.Context, spinner *utils.Spinner, s *model.St
 		if err := services.Destroy(ctx, dList[i].Name, dList[i].Namespace, c); err != nil {
 			return fmt.Errorf("error destroying service '%s': %s", dList[i].Name, err)
 		}
-		spinner.Stop()
 		if _, ok := s.Services[dList[i].Name]; ok {
 			oktetoLog.Success("Destroyed previous service '%s'", dList[i].Name)
 		} else {
 			oktetoLog.Success("Service '%s' destroyed", dList[i].Name)
 		}
-		spinner.Start()
```

Comment:
Why are we removing the spinner in some functions and not in other?

Reply:
same here https://github.com/okteto/okteto/pull/2994#discussion_r944259505